src/Select/StyledSelect.tsx

```diff
@@ -35,26 +35,12 @@ const StyledSelectWrapper = styled.div`
   }
 `;
 
-const StyledSelect = ({ children, ...props }) => {
-  const { large } = props;
-
+const StyledSelect: FC = ({ children, ...props }) => {
   return (
-    <StyledSelectWrapper large={large}>
```

Comment:
This didn't break any snapshots, and I see no evidence that `large` is meaningful for `StyledSelectWrapper`.